packages/app/src/sandbox/eval/transpilers/svelte/svelte-worker.js

```diff
@@ -1,21 +1,22 @@
+import semver from 'semver';
 import { buildWorkerError } from '../utils/worker-error-handler';
 import { buildWorkerWarning } from '../utils/worker-warning-handler';
 
 // Allow svelte to use btoa
 self.window = self;
 
-self.importScripts(['https://unpkg.com/svelte@^2.0.0/compiler/svelte.js']);
+self.importScripts(['https://unpkg.com/svelte@3.0.0/compiler.js']);
 
 self.postMessage('ready');
 
-declare var svelte: {
-  compile: (code: string, options: Object) => { code: string },
-};
+// declare var svelte: {
+//   compile: (code: string, options: Object) => { code: string },
+// };
 
 function getV2Code(code, path) {
   const {
     js: { code: compiledCode, map },
-  } = svelte.compile(code, {
+  } = window.svelte.compile(code, {
```

Comment:
I'd make this `self`, we're in the context of a web worker and they normally don't have `window` (except this time because of our hack at the start of the file)

Reply:
Updated